vuepress/docs/next/docs/component-generator/blueprint-features.md

```diff
@@ -0,0 +1,135 @@
+# Entando Blueprint Features
+
+The Entando Blueprint makes it easier and faster to customize your application by   generating controllers, repositories, services, and micro frontends for your entity. It means generating those project files in minutes simply by answering questions that define the parameters of your applications. 
+
+The Entando Blueprint uses JHipster Technology based on Embedded JavaScript (EJS), a templating language that provides powerful constructs for large scale file generation. 
+
+## The Features:
+* Backend with Spring Boot 
+  * profiles (dev, prod)
+  * Pre-configured Cross-Origin Resource Sharing (CORS) settings
+  * Keycloak integration 
+  * Swagger/OpenAPI frontend
+  * Data modeling 
+  * Jhipster Design Language (JDL) support for schema upgrade
+  * Liquibase integration for schema upgrade
+* Frontend 
+  * Localization
+  * Keycloak integration
+  * .env profiles 
+
+For more information:
+* Install [Jhipster Entando Blueprint](https://github.com/entando/generator-jhipster-entando/blob/master/README.md)
+* Try implementing an Entando Blueprint-- [Create micro services and micro Frontents](../../tutorials/micro-frontends/react.html)
+
+## Premade Widgets
+When you create an entity using the Entando Blueprint, it generates a few
+premade widgets. Below, each will be reviewed.
+* [Authentication](#authentication)
+* [Custom events](#custom-events)
+* [Tests and mocks](#tests-and-mocks)
+* [PropTypes](#proptypes)
+* [Fetching data](#fetching-data)
+* [Form widget](#form-widget)
+
+We will be using `Conference` as an entity name for the example below.
+
+What the widgets have in **common**:
+* Each widget contains a README file that helps with the setup.
+
+* All generated widgets are web components created using Custom Elements
```

Comment:
"All widgets are web components created with the Custom Elements API."

Reply:
we are only talking about the generated widgets